tests/cli-tester.rs

```diff
@@ -61,9 +61,9 @@ fn test_no_clobber() {
         .arg("--language")
         .arg("json")
         .arg("--input-file")
-        .arg(&input_path)
+        .arg(input_path)
```

Comment:
Clippy isn't warning me about this, for some reason. The reference is because `input_path` used to be a `Path`, rather than `&Path`, FWIW... Is there some configuration to make Clippy more vocal about these kinds of things?

Reply:
My guess is you're on an older version:

```
cargo --version
cargo 1.67.1 (8ecd4f20a 2023-01-10)
```